denis.roman/task1.rb

```diff
@@ -0,0 +1,6 @@
+Arr = Array.new(3) { Array.new(3) { rand -10...10 } }
+p Arr
+positive_numbers=Arr.flatten.select(&:positive?) 
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#spaces-braces
Please take a look on a bbatsov's ruby style guide, there are a lot of tips so your code can become more readable
https://github.com/rubocop-hq/ruby-style-guide